packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -2324,3 +2320,28 @@ function isTruthyOrNull(obj: any): boolean {
 function isUndefined(obj: any): boolean {
   return obj === undefined;
 }
+
+interface IndexedRules {
```

Comment:
nit: We could move these types into `AuthRule.ts` aside from the negate function.

Reply:
Sure